test/team_test.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'test_helper'
+require './lib/team'
+
+class TeamTest < Minitest::Test
+
+  def setup
+    @atlanta = {
+      team_id: "1",
+      franchise_id: "23",
```

Comment:
we forgot to change the franchise_id, it was still franchiseID, so renamed these to match team class

Reply:
whoever didn't rename franchiseID should be taken out in public and shamed.